src/Markdig.Tests/Specs/Specs.cs

```diff
@@ -17582,8 +17582,8 @@ public void Example003()
             //     +---------+---------+---------+
             //     | Col1    | Col2    | Col3    |
             //     | Col1a   | Col2a   | Col3a   |
-            //     | Col12             | Col3b   |
-            //     | Col123                      |
+            //     | Col1b             | Col3b   |
```

Comment:
D'oh!
Navigated to the specs.cs file from ReSharper test session window, didn't even notice it was generated from T4!